spec/controllers/order_management/order_details_controller_spec.rb

```diff
@@ -495,7 +495,17 @@ def reservation_params(reserve_start_at, actual_start_at = nil)
         expect(order_detail.reload.actual_total).to eq(16.00)
       end
 
-      it "returns an error when trying to set subsidy more than quantity" do
+      xit "updates the price while changing accounts" do
```

Comment:
I buy that the formal `pending` makes sense. It hopefully will be short-lived, but I'm not sure when we'll be able to get to that piece.